test/e2e/cronjob_test.go

```diff
@@ -56,7 +59,57 @@ func TestSourceCronJob(t *testing.T) {
 	jpSinkRefNameInSpec := "jsonpath={.spec.sink.ref.name}"
 	out, err := test.getResourceFieldsWithJSONPath("cronjobsource", "testcronjobsource2", jpSinkRefNameInSpec)
 	assert.NilError(t, err)
-	assert.Equal(t, out, "testsvc1")
+	//assert.Equal(t, out, "testsvc1")
+	assert.Check(t, util.ContainsAllIgnoreCase(out, "testsvc1"))
+
+	t.Log("verify event messages from cronjob source")
+	mymsg := "This is a message from cronjob."
+	test.serviceCreateEventDisplay(t, r, "displaysvc")
+	test.cronJobSourceCreate(t, r, "testcronjobsource3", "*/1 * * * *", mymsg, "svc:displaysvc")
+	results := test.kn.Run("source", "cronjob", "describe", "testcronjobsource3")
+	r.AssertNoError(results)
+	out, err = kubectl{test.kn.namespace}.Run("get", "pod", "-l", "sources.eventing.knative.dev/cronJobSource=testcronjobsource3", "-o", "yaml")
+	t.Log(out)
```

Comment:
Is this necessary ? Maybe better assert on `out`.

Reply:
for test purpose. will remove in the latest version.